### PR TITLE
feature(tj): add monitoring for the gradient conflict metric of MoE in ScaleZero

### PR DESCRIPTION
**Pull request by tAnGjIa520**

### CODE REVIEW

File: lzero/entry/train_unizero_multitask_segment_ddp.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 from functools import partial
 from typing import Tuple, Optional, List
+import concurrent.futures
```
Comment (puyuan1996) [inline lzero/entry/train_unizero_multitask_segment_ddp.py:5]: 需要merge [dev-multitask-balance-clean](https://github.com/opendilab/LightZero/tree/dev-multitask-balance-clean) 解决冲突

Comment (tAnGjIa520) [reply lzero/entry/train_unizero_multitask_segment_ddp.py:5]: 已解决，这个MoE冲突的应该是单独开一个分支的吧

File: toy/multitask_gating_experiment_version.py [hunk 2]
```diff
@@ -0,0 +1,1501 @@
+import torch
+import torch.nn as nn
```
Comment (puyuan1996) [inline toy/multitask_gating_experiment_version.py:2]: 这个文件是和 https://github.com/opendilab/LightZero/pull/401/files 一致的吗？如果是改一下文件name 以及移到lzero/model/unizero_world_models/toy_multitask_moe_grad_analysis.py这个路径下吧

File: lzero/policy/unizero_multitask.py [hunk 3]
```diff
@@ -422,7 +435,7 @@ def _init_learn(self) -> None:
             device_type=self._cfg.device,
             betas=(0.9, 0.95),
         )
-
+        # self.a=1
```
Comment (puyuan1996) [inline lzero/policy/unizero_multitask.py:438]: 删除

File: lzero/entry/train_unizero_multitask_segment_ddp.py (hunk 1 above)
Comment (puyuan1996) [reply lzero/entry/train_unizero_multitask_segment_ddp.py:5]: 这个PR后面应该是放在单独的分支哈，因为与主逻辑差别较大。

File: lzero/model/common.py [hunk 4]
```diff
@@ -248,6 +248,68 @@ def remove_hooks(self):
         self.forward_handler.remove()
         self.backward_handler.remove()
 
+# # modified by tangjia
```
Comment (puyuan1996) [inline lzero/model/common.py:251]: 这里的代码有用到吗 需要删除吧？

File: zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py [hunk 5]
```diff
@@ -333,7 +333,7 @@ def create_env_manager():
 
         torchrun --nproc_per_node=8 ./zoo/atari/config/atari_unizero_multitask_segment_8games_ddp_config.py
     """
-
+# /fs-computility/niuyazhe/tangjia/code/LightZero-dev-multitask-balance-clean/zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py
```
Comment (puyuan1996) [inline zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py:336]: 删除

File: zoo/atari/config/atari_unizero_multitask_segment_ddp_config_debug.py [hunk 6]
```diff
@@ -64,8 +64,8 @@ def create_config(env_id, action_space_size, collector_env_num, evaluator_env_nu
         policy=dict(
             multi_gpu=True,  # Very important for ddp
             only_use_moco_stats=False,
-            # use_moco=False,  # ==============TODO==============
-            use_moco=True,  # ==============TODO: moco==============
+            use_moco=False,  # ==============TODO==============
```
Comment (puyuan1996) [inline zoo/atari/config/atari_unizero_multitask_segment_ddp_config_debug.py:67]: 实际scalezero moe grad分析这里是用的这个还是上面那个config？只保留使用的那个，以及写一个英文和中文 readme整体介绍一下这个PR的相关改动和moe grad分析结果的大致情况，放在config同等路径下

**Review by puyuan1996 — CHANGES_REQUESTED** [review]

File: zoo/atari/config/atari_unizero_multitask_segment_ddp_config_debug.py (hunk 6 above)
Comment (tAnGjIa520) [reply zoo/atari/config/atari_unizero_multitask_segment_ddp_config_debug.py:67]: use_moco=False,  这一个

File: lzero/policy/unizero_multitask.py (hunk 3 above)
Comment (tAnGjIa520) [reply lzero/policy/unizero_multitask.py:438]: 已解决，类似的已去除

File: zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py (hunk 5 above)
Comment (tAnGjIa520) [reply zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py:336]: 类似的已删除
